### DISCUSSION TITLE
What to do if some AWS services are missing from Prowler scan such as Kinesis and SES

### DISCUSSION THREAD

**Discussion by qadri99-max** [prowler-cloud/prowler#5614] [Q&A]
I ran `'prowler aws --list-services'` to get a list of services and noticed that some are missing.

I ran `'prowler aws --services kinesis' `anyway to see what happens, and I think Prowler just hangs.

Is there a way to scan services not currently listed? 
thanks

**Comment by drewkerrigan** [comment]
Hello! I did a little research and found that the following services are missing from the `prowler aws --list-services` command output:

```
- bedrock
- datasync
- directconnect
- elasticbeanstalk
- fsx
- globalaccelerator
- kinesis
- mq
- ses
- waf
```

I've reached out to the team to see if there's a good reason for those to be omitted, and we'll follow up soon. Thanks for the reporting this!

**Comment by jfagoagas** [comment]
Hello @qadri99-max, what version of Prowler are you using? Some of them are new and not yet released.

In the `master` branch I can see all of these services except `globalaccelerator` since it has no checks, because a service is only listed if has one or more checks. In this particular case it is used in another service.

**Reply by qadri99-max** [reply]
Prowler version is 4.4.0

I am creating a Docker image that contains the following code:
`RUN git clone https://github.com/toniblyx/prowler`

Now I wonder if I should be using this url: git clone https://github.com/**prowler-cloud**/prowler

**Reply by jfagoagas** [reply]
Hello @qadri99-max if you are cloning the Prowler repository, the `main` branch has the v4.5.0 which is not yet released and there you should be able to get all the services available. Could you please try that?

Thanks!
